packages/ui-components/src/components/top-bar.ts

```diff
@@ -0,0 +1,28 @@
+import { html } from 'lit';
+import { BaseElement } from '../base';
+import { property } from 'lit/decorators.js';
+
+export class TopBar extends BaseElement {
+  @property({ type: String })
+  logo = '';
+
+  getLogo() {
+    if (this.logo) {
+      return html`<img class="p-2 h-12" src="${this.logo}" />`;
+    } else {
+      return html``;
+    }
+  }
```

Comment:
This function can be removed, a simple ternary in combination with `nothing` should suffice.

Reply:
fixed in: https://github.com/JortWillemsen/stryker-dashboard-redesign/pull/20